.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: emacs-lisp
+sudo: false
+cache: apt
+env:
+  - EVM_EMACS=emacs-25.1-travis
+  - EVM_EMACS=emacs-25.2-travis
```

Comment:
EVM has an `emacs-git-snapshot-travis` build but it seems like maybe not the best plan to have our tests dependent on unstable behavior in the emacs git repo? It also seems unnecessary. On second thought I don't really care very much and this is unlikely to break anything. I'm happy to go with whatever you think is the best plan.